web/src/components/tasks/business-formation/business/BusinessNameAndLegalStructure.tsx

```diff
@@ -45,14 +45,14 @@ export const BusinessNameAndLegalStructure = ({ isReviewStep = false }: Props):
 
   const legalStructureName = (): string => {
     if (!business || !business.profileData.legalStructureId) return "";
-    const preface =
-      business?.profileData.businessPersona === "FOREIGN"
-        ? `${Config.formation.legalStructure.foreignPrefaceText} `
-        : "";
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    const legalStructure = (Config.formation.legalStructure as any)[business.profileData.legalStructureId];
-    return `${preface} ${legalStructure}`;
+    if (business.profileData.businessPersona === "FOREIGN") {
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      return (Config.formation.legalStructure.foreignLabels as any)[business.profileData.legalStructureId];
```

Comment:
It'd be cool if we didn't need `as any`, but I don't know if that's worth the effort. This is fine.